### PR TITLE
Enhance README

### PR DESCRIPTION
**Pull request by petroav**
* Add installation and deployment instructions.
* Add requirements section.
* Add developers section.

### CODE REVIEW

**Comment by petroav** [comment]
@anusudarsan & @kokosing can one of you review when you get a chance? Thanks!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+## <a name="iifu"></a>Integration instructions for users
+
+Unfortunately, at the moment Ambari does not support a more user friendly installation method. The installation has to be done by following the somewhat manual steps outlined below.
+
+1. Create the distribution by executing `make clean dist`. This will create a `tar.gz` in the dist directory. This file is the distribution you'll need to upload to your cluster.
+2. Upload the distribution archive to the node in your cluster that runs the Ambari master server.
+3. Assuming that `ambari-server` is installed in `/var/lib/ambari-server`, you should execute the following:
+```
+mkdir /var/lib/ambari-server/resources/stacks/HDP/2.2/services/PRESTO
+tar -xvf ambari-presto-0.1.0.linux-x86_64.tar.gz -C /var/lib/ambari-server/resources/stacks/HDP/2.2/services/PRESTO
```
Comment (anusudarsan) [inline README.md]: by default both presto n ambari use port 8080 right?

**Comment by anusudarsan** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (petroav) [inline README.md]: Yes, I'm impressed you caught that :) That's why by default in config.properties the port for Presto is set to 8081.

**Pull request merged**